src/Stratis.CirrusPegD/Program.cs

```diff
@@ -78,8 +78,11 @@ private static IFullNode GetMainchainFullNode(string[] args)
                 MinProtocolVersion = ProtocolVersion.ALT_PROTOCOL_VERSION
             };
 
+            NetworkType networkType = nodeSettings.Network.NetworkType;
+
             var fedPegOptions = new FederatedPegOptions(
-                counterChainNetwork: SidechainNetworks[nodeSettings.Network.NetworkType]()
+                counterChainNetwork: SidechainNetworks[networkType](),
+                walletSyncFromHeight: new int[] { 1, 1, 1 }[(int)networkType] // TODO
```

Comment:
Happy with `1, 1, 1_100_000` ? Or can reuse the constant value in FederationGatewaySettings

Reply:
I think that constant is meant to indicate when the first block was mined and not necessarily when the wallet was created. But then again it should be the first transaction in the wallet right?

I will make the change.